sw/otbn/meson.build

```diff
@@ -0,0 +1,102 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# Build definitions for OTBN software
+#
+# OTBN software is built with a separate toolchain, which is called as an
+# external target from meson. All functionality to call the external toolchain
+# is encapsulated in this build file; users only need to care about two
+# dictionaries.
+#
+# To use an OTBN application in device software, use the sw_otbn dictionary.
+# For example, to add a dependency to an embeddable version of the barrett384
+# OTBN application, add the following variable to the list of dependencies:
+# sw_otbn['barrett384']['rv32embed_dependency']
```

Comment:
Perhaps the second string wants to be a constant?

Reply:
I feel the current string is sufficiently safe and descriptive.